CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ if(NOT TARGET manpages)
   add_custom_target(manpages ALL)
 endif()
 
-find_package(VOTCA_TOOLS REQUIRED NO_MODULE)
+find_package(VOTCA_TOOLS NO_MODULE REQUIRED)
```

Comment:
why that?

Reply:
Umh... how about become I don't know what I'm doing with cmake and the documentation and examples are lacking even though its been around for years.